int8 count
---
geometry_msgs/Pose pose
